specification/mcpp/resource-manager/Microsoft.CloudPartnerProgramMembership/preview/2023-02-09-preview/examples/getMcppPurchaseDetailsList.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
+    "resourceGroupName": "McppPurchases",
+    "api-version": "2023-02-09-preview",
+    "subscriptionId": "291bba3f-e0a5-47bc-a099-3bdcb2a50a05"
+  },
+  "responses": {
+    "200": {
+      "body": {
+          "value": [ 
+            { 
+              "id": "microsoftactionpack",
```

Comment:
id should always be a full path on which one can do a GET. So:

```
"id": "/subscriptions/......../providers/Microsoft.CloudPartnerProgram/memberships/microsoftactionpack"
```

Reply:
ack, will change this.